random-circle-generator/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html dir="ltr">
+	<head>
+		<title>es6 comperhand</title>
+		<meta charset="utf-8">
+		<style>
+			.speed{
+				position:absolute;
+				top:8px;
+				right:0;
+			}
+			#count{
+				display:inline        
+			}
+      #root {        
+      }
+		</style>
+	</head>
+
+  <body style="background:rgb(163, 163, 163)">  
+    
+    <div id="root"></div>				
+    
+	</body>
+</html>
```

Comment:
Any specific reason for that indentation? :eyes:

Reply:
You can use [EditorConfig](https://editorconfig.org/) to help you with that. It helps a lot to keep your code style consistent between everyone working on the same project. You just have to install it on your editor and add a `.editorconfig` file, the most commonly used seems like this:

```
# EditorConfig helps developers define and maintain consistent
# coding styles between different editors and IDEs
# http://editorconfig.org

# top-most EditorConfig file
root = true

[*]
charset = utf-8
end_of_line = lf
indent_style = space
indent_size = 2
trim_trailing_whitespace = true
insert_final_newline = true
```